src/tests/test_api.py

```diff
@@ -15,6 +16,11 @@
 
 from .test_process import TestProcessProto
 
+sample_statusbar_msg = "new status bar message"
+sample_inbox_msg_id = '2c975c515f8b414db5eea60ba57ba455'
+sample_inbox_msg_receiver_address = 'BM-2cWzSnwjJ7yRP3nLEWUV5LisTZyREWSzUK'
+sample_trash_msg_ids = ['27e644765a3e4b2e973ee7ccf958ea20', '51fc5531-3989-4d69-bbb5-68d64b756f5b']
+sample_test_subscription_address = ['BM-2cWQLCBGorT9pUGkYSuGGVr9LzE4mRnQaq', 'BM-GtovgYdgs7qXPkoYaRgrLFuFKz1SFpsw']
```

Comment:
There is already a module `tests.samples` for this: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/tests/samples.py